tasks/gen.go

```diff
@@ -90,10 +90,10 @@ func main() {
 		schema.SpawnStructField("Miner", "String", false, false),
 		schema.SpawnStructField("PayloadCID", "String", false, false),
 		schema.SpawnStructField("CARExport", "Bool", false, false),
-		schema.SpawnStructField("Schedule", "String", true, false),
-		schema.SpawnStructField("ScheduleLimit", "String", true, false),
-		schema.SpawnStructField("Tag", "String", true, false),
-		schema.SpawnStructField("MaxPriceAttoFIL", "Int", true, false),
+		schema.SpawnStructField("Schedule", "String", true, true),
+		schema.SpawnStructField("ScheduleLimit", "String", true, true),
+		schema.SpawnStructField("Tag", "String", true, true),
+		schema.SpawnStructField("MaxPriceAttoFIL", "Int", true, true),
```

Comment:
I assume we're all in agreement this is a backwards compatible change in terms of encoded data, since we simply add an extra option for these fields to be `null`. Just thinking about it outloud.

Reply:
Ah I see. I replied to your ipld-prime thread. We already have live data, so I agree with your fix here - the alternative would have been to use Representation consistently with codecs.